experiments/download_ssl4eo.py

```diff
@@ -576,7 +460,7 @@ def worker(idx: int) -> None:
         # add to existing checked locations
         with open(ext_path, "a") as f:
```

Comment:
It's a shame we have to open and close a file for every single patch, but I don't see a better way to do it if we want to support parallelism.

Reply:
yeah...